### PR TITLE
feat(Tubi): multilang, buttons, covers, privacy & refactor

### PR DESCRIPTION
**Pull request by DarkVillager**
## Description 

Added Buttons, Covers, Privacy & Multi lang with settings, added live video & refactored
## Acknowledgements
- [X] I read the [Presence Guidelines](https://github.com/PreMiD/Presences/blob/main/.github/CONTRIBUTING.md)
- [x] I linted the code by running `yarn format`
- [X] The PR title follows the repo's [commit conventions](https://github.com/PreMiD/Presences/blob/main/.github/COMMIT_CONVENTION.md)

## Screenshots
<details>
<summary> Proof showing the creation/modification is working as expected </summary>

</details>

### CODE REVIEW

File: websites/T/Tubi/presence.ts [hunk 1]
```diff
@@ -1,41 +1,101 @@
...
+			presenceData.buttons = [
+				{
+					label: "View Video",
+					url: href,
+				},
+			];
+		} else {
+			presenceData.buttons = [
+				{
+					label: "View Live Video",
```
Comment (EncryptedDev) [inline websites/T/Tubi/presence.ts:82]:
```suggestion
					label: "Watch Live",
```

File: websites/T/Tubi/presence.ts [hunk 2]
```diff
@@ -1,41 +1,101 @@
...
+			presenceData.smallImageText = video.paused
+				? strings.paused
+				: strings.play;
+			presenceData.endTimestamp = presence.getTimestamps(
+				Math.floor(video.currentTime),
+				Math.floor(video.duration)
+			)[1];
+			presenceData.buttons = [
+				{
+					label: "View Video",
```
Comment (EncryptedDev) [inline websites/T/Tubi/presence.ts:75]:
```suggestion
					label: "Watch Video",
```

**Review by EncryptedDev — CHANGES_REQUESTED** [review]

**Review by EncryptedDev — APPROVED** [review]

File: websites/T/Tubi/presence.ts [hunk 3]
```diff
@@ -1,41 +1,101 @@
...
+			presenceData.smallImageText = video.paused
+				? strings.paused
+				: strings.play;
+			presenceData.endTimestamp = presence.getTimestamps(
+				Math.floor(video.currentTime),
+				Math.floor(video.duration)
+			)[1];
+			presenceData.buttons = [
+				{
+					label: "Watch Video",
```
Comment (Bas950) [inline websites/T/Tubi/presence.ts:75]: No string for that?

File: websites/T/Tubi/presence.ts [hunk 4]
```diff
@@ -1,41 +1,101 @@
...
+					url: href,
+				},
+			];
+		} else {
+			presenceData.buttons = [
+				{
+					label: "Watch Live",
```
Comment (Bas950) [inline websites/T/Tubi/presence.ts:82]: Same

File: websites/T/Tubi/presence.ts [hunk 5]
```diff
@@ -1,41 +1,101 @@
...
+	if (privacy) {
+		presenceData.details = strings.browse;
+		return;
+	}
+	if (search?.value) {
+		presenceData.details = "Searching for";
+		presenceData.state = search.value;
+		presenceData.smallImageKey = Assets.SearchImage;
+	} else if (pathname.includes("/category/")) {
+		presenceData.details = "Viewing category";
```
Comment (Bas950) [inline websites/T/Tubi/presence.ts:53]: Same

File: websites/T/Tubi/presence.ts [hunk 6]
```diff
@@ -1,41 +1,101 @@
...
+		oldLang = newLang;
+		strings = await getStrings();
+	}
+
+	if (privacy) {
+		presenceData.details = strings.browse;
+		return;
+	}
+	if (search?.value) {
+		presenceData.details = "Searching for";
```
Comment (Bas950) [inline websites/T/Tubi/presence.ts:49]: Same

**Review by Bas950 — CHANGES_REQUESTED** [review]

File: websites/T/Tubi/presence.ts (hunk 6 above)
Comment (RisingSunLight42) [reply websites/T/Tubi/presence.ts:49]: Exist in general strings :
https://github.com/PreMiD/Localization/blob/main/src/Presence/general.json#L106-L107

File: websites/T/Tubi/presence.ts (hunk 3 above)
Comment (RisingSunLight42) [reply websites/T/Tubi/presence.ts:75]: Exist in general strings :
https://github.com/PreMiD/Localization/blob/main/src/Presence/general.json#L318-L320

File: websites/T/Tubi/presence.ts (hunk 4 above)
Comment (RisingSunLight42) [reply websites/T/Tubi/presence.ts:82]: Similar in general strings :
https://github.com/PreMiD/Localization/blob/main/src/Presence/general.json#L322-L324

File: websites/T/Tubi/presence.ts (hunk 5 above)
Comment (RisingSunLight42) [reply websites/T/Tubi/presence.ts:53]: Exist in general strings :
https://github.com/PreMiD/Localization/blob/main/src/Presence/general.json#L150-L151

**Review by Bas950 — APPROVED** [review]

**Pull request merged**